docs/guide/plugins.md

```diff
@@ -91,6 +98,38 @@ Two more functions are currently been added to the API in a [Pull Request for ha
 
 Future progress on the Plugin API will be reported here. Help is always welcome.
 
+### Using plugins to define shortcuts for ui actions
+
+Currently Xournal++ does not have shortcuts/keybindings configurable in the preferences. However you can write your custom plugin to achieve exactly that.
+
+The function `app.uiAction` used for simulating a toolbar/menubar click can be used for lots of different actions. 
+This command accepts a Lua table with keys
+`"action"`, `"group"` and `"enabled"`
+as its argument. The `"group"` key is only used for displaying warnings, so you can omit it. The `"enabled"` key is set to `true` by default, so you can often omit it as well. The `"action"` key accepts
+one of the following values:
+
+- `"ACTION_NEW", "ACTION_OPEN", "ACTION_ANNOTATE_PDF", "ACTION_SAVE", "ACTION_SAVE_AS", "ACTION_EXPORT_AS_PDF", "ACTION_EXPORT_AS", "ACTION_PRINT", "ACTION_QUIT"` for **file menu** actions
```

Comment:
I feel like it would be better to just link to the action source file, otherwise this can become outdated easily. The exact actions that can be used will be tied to Xournal++ version.

Reply:
OK, I agree.